### PR TITLE
Refactor /management/index.js to functional component

### LINKED ISSUE #8676: [UI] Convert `/management/index.js` from class based functional component
### Current Behavior
`/management/index.js` is a class-based component. 

### Desired Behavior
`/management/index.js` should be a functional component 

### Implementation
Here is a quick blog that will help you get started with : [Link](https://www.digitalocean.com/community/tutorials/five-ways-to-convert-react-class-components-to-functional-components-with-react-hooks)

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by dhiraj0911**
Notes for Reviewers

#### Refactored /management/index.js from a class-based component to a functional component using React hooks.

This PR fixes
#8676 

Changes Made:

- Imported useEffect from React.
- Converted Index from a class component to a functional component.
- Replaced componentDidMount with the useEffect hook.
- Removed the render method and directly returned the JSX.
- **[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**

- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/package-lock.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
         "path-parse": "^1.0.7",
         "pluralize": "^8.0.0",
         "postcss": "^8.4.21",
-        "prop-types": "*",
+        "prop-types": "latest",
```
Comment (Yashsharma1911) [inline ui/package-lock.json:66]: Why these changes @dhiraj0911

Comment (dhiraj0911) [reply ui/package-lock.json:66]: It's an auto-generated, I'll revert it back to previous state

**Comment by dhiraj0911** [comment]
@Yashsharma1911 can you review this again

**Review by Yashsharma1911 — APPROVED** [review]
Thankyou LGTM, @Rajdip019 thoughts?

**Review by Rajdip019 — APPROVED** [review]
LGTM. Amazing work @dhiraj0911

**Comment by dhiraj0911** [comment]
Thank you @Yashsharma1911 @Rajdip019

**Comment by dhiraj0911** [comment]
Are we good to merge?
@Yashsharma1911 @Rajdip019

**Comment by Rajdip019** [comment]
> Are we good to merge? @Yashsharma1911 @Rajdip019

LGTM @Yashsharma1911

**Comment by Yashsharma1911** [comment]
Thank you gents
